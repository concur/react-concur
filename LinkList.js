import React from 'react';
import Link from './Link';

export default class LinkList extends React.Component {
  render() {
    return (
      <div>
        {this.props.hrefs.map(href => {
          return <Link key={href} page={href} />
        })}
      </div>
    );
    
  }

}