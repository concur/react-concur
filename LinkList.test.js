import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import LinkList from './LinkList';

test('renders Links when passed an array of hrefs', () => {
  // Render a checkbox with label in the document
  const linkList = shallow(
    <LinkList hrefs={['yahoo', 'google']} />
  );
  expect(toJson(linkList)).toMatchSnapshot();
  
});