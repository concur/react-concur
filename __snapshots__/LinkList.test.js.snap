// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Links when passed an array of hrefs 1`] = `
<div>
  <Link
    page="yahoo"
  />
  <Link
    page="google"
  />
</div>
`;
